contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -350,13 +346,10 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         // Store the receipt of this message delivery. When a subsquent message is sent back
         // to the origin of this message, we will clean up the receipt state.
         // If the receipts queue contract for this chain doesn't exist yet, create it now.
-        ReceiptQueue receiptsQueue = outstandingReceipts[
+        ReceiptQueue.TeleporterMessageReceiptQueue storage receiptsQueue = outstandingReceipts[
```

Comment:
What happens in this case if the mapping does not exist?

Reply:
IIUC since the `outstandingReceipts` is a mapping to structs instead of `ReceiptQueue` contract instances now, all chain ids will have the default mapping exist. Will want to make sure to have this tested.

Reply:
can we add a unit test covering this case, and making sure it defaults to all the zero values for chains that we have not received from.

Reply:
I think [this test](https://github.com/ava-labs/teleporter/blob/main/contracts/src/Teleporter/tests/ReceiveCrossChainMessageTests.t.sol#L24) covered this case.

Reply:
can we add a specific test case for this in terms of:
- create a `mapping` similar to `outstandingReceipts`
- grab a receipt queue for a mock chain id
- go through every value in the queue struct and check defaults to 0